src/components/Footer.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled, { css } from 'styled-components';
+import { ReactComponent as FooterLogo } from '../assets/sign.svg';
+
+const FooterWrapper = styled.footer`
+  display: flex;
+  align-items: center;
+  position: absolute;
+  bottom: 0;
+  height: 100px;
+  width: 100%;
+`;
+
+const Nav = styled.nav`
+  display: flex;
+  width: 100%;
+  margin: auto;
+  justify-content: space-between;
```

Comment:
`space-between` doesn't really work here. The problem is (as you realized) that the elements have the same space in between them. But since they have different widths they are not evenly spread (I mean being on the left/center/right). That's why you need to use the margin on the logo.

An alternative is to set `flex: 1` to the links. This way the links will take up all space and the logo will only take what it requires.

Reply:
I was trying to figure out how to do that for so long!  That's when I ended up doing the margin.  It felt hacky but I wasn't sure what to do at that point.  Thanks for the heads up!